src/models/generation.jl

```diff
@@ -13,3 +18,13 @@ function IS.get_limits(
     # e.g. "valid_range": {"min":-1.571, "max":1.571}.
     return (min = valid_range.min, max = valid_range.max, zero = 0.0)
 end
+
+"""
+Return the max active power for the Renewable Generation calutades as the rating * powe_factor
+"""
+function get_max_active_power(d::T) where T <: RenewableGen
+    if !hasmethod(get_rating, T) || !hasmethod(get_power_factor, T)
```

Comment:
I think you're better off letting Julia fail on its own (certainly more performant).

If you disagree, I think your throw statement is wrong.  I think you meant `get_max_active_power` instead of `get_max_reactive_power`.  Second, are you sure that syntax is correct?

Reply:
I think there shouldn't be too much of a performance hit to run these checks. I will correct the typo

Reply:
I think you have syntax errors.  Do the tests pass?

Reply:
3dd11b4